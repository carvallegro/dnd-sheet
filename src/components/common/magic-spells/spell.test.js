import React from 'react'
import { shallow } from 'enzyme'

import Spell from './spell'

describe('<Spell />', () => {
  it('renders', () => {
    const wrapper = shallow(<Spell />)
    expect(wrapper).toMatchSnapshot()
  })
})
