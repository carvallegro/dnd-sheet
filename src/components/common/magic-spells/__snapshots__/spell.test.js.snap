// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spell /> renders 1`] = `
<styled.div>
  <Checkbox
    name="spell-a"
    onChange={[Function]}
    size="xsmall"
  />
  <Styled(styled.input)
    size="small"
  />
</styled.div>
`;
